### PR TITLE
Put copies number

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

File: object/service.proto [hunk 1]
```diff
@@ -84,6 +84,8 @@ message PutRequest {
         Object Object       = 1;
         // Token with session public key and user's signature
         session.Token Token = 2;
+        // Number of the object copies to store within the RPC call (zero is processed according to the placement rules)
+        uint32 CopiesNumber = 3;
```
Comment (im-kulikov) [inline object/service.proto:88]: mb we can be named it like in the container service? `ReplFactor`

Comment (realloc) [reply object/service.proto:88]: The meaning is different here. Technically it's a number of ACKs from Storage nodes. Let it be Copies here to simplify understanding.

Comment (im-kulikov) [reply object/service.proto:88]: thanks for explaining

Comment (cthulhu-rider) [reply object/service.proto:88]: @im-kulikov, explanations from @realloc were ahead of mine.
@realloc, clarify, please, do you agree w/ `CopiesNumber` or propose to trim it to `Copies`.

Comment (realloc) [reply object/service.proto:88]: `CopiesNumber` is fine

**Pull request merged**